include/core/CRapidJsonLineWriter.h

```diff
@@ -29,7 +29,7 @@ template<typename OUTPUT_STREAM,
          typename TARGET_ENCODING = rapidjson::UTF8<>,
          typename STACK_ALLOCATOR = rapidjson::CrtAllocator,
          unsigned WRITE_FLAGS = rapidjson::kWriteDefaultFlags>
-class CORE_EXPORT CRapidJsonLineWriter
```

Comment:
Please can you also remove the header from line 12.

Reply:
Done

Reply:
Done